course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,164 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){}
+
+MyString::MyString(const char* argString)
+{
+        buffer = std::unique_ptr<char[]>(new char[ std::strlen(argString) + 1]);
+        std::strcpy(buffer.get(), argString);;
```

Comment:
I'll have to check my XCode settings, but I'm pretty sure it was set to warn. But there were no warnings when compiling in the IDE.